### ISSUE TITLE
multiple go projects in one go root

### ISSUE THREAD

**Issue by dawi** [go-lang-plugin-org/go-lang-idea-plugin#698]
There are great sources out there how to structure go code. For Example:
http://golang.org/doc/code.html#Organization
http://peter.bourgon.org/go-in-production

The idiomatic go approach of structuring projects is to have one gopath that contains packages / projects.

The intention is obviously to have a simple and standardised way to manage go code. And it is really easy to follow this principles as long as you are just using one of your favorite editors and the command line.

As a go beginner and long time IntelliJ IDEA user I wanted to use the go plugin for my go projects. My observations are: On the one hand, it is really easy to start a project from scratch. On the other hand, using the plugin, makes it really hard to follow the idiomatic go approach. 

Let me give an example:

Given a company with many very different projects.
Each project needs slightly different run configurations, build-scripts, etc.
It is only natural to make multiple idea projects of it.

As a result the gopath structure would look like this:

```
src/
    github.com/nf/
        streak/
            .git/                  # git repository metadata
            .idea/                 # idea project folder
            oauth.go               # command source
            streak.go              # command source
        todo/
            .git/                  # git repository metadata
            .idea/                 # idea project folder
            task/
                task.go            # package source
            todo.go                # command source
```

Right now it does not seem to be possible.

Are there any things I am missing or am I wrong at some point?

Regards, 
Daniel

**Comment by dlsniper** [comment]
Fixed in 1.0
